cli/tests/fixtures/extensions/permissions/incorrect-run-perms/PhylumExt.toml

```diff
@@ -0,0 +1,3 @@
+name = "incorrect-run-perms"
+description = "An extension with correctly specified permissions"
```

Comment:
As far as I can tell, this extension isn't actually called from any tests.

Additionally, the description and name are in conflict (`correctly specified permissions` vs. `incorrect-run-perms`)

Reply:
I recently added a new test and contemplated if it was worth to add a fixture and decided against it myself.

Imho it's just additional overhead at this point to check the extension fixture directory rather than having everything relevant defined right in the test.

Reply:
> Imho it's just additional overhead at this point to check the extension fixture directory rather than having everything relevant defined right in the test.

I agree. So this `incorrect-run-perms` extension is going away?